python/metatensor-learn/metatensor/learn/__init__.py

```diff
@@ -0,0 +1,11 @@
+import sys
+
+if (sys.version_info.major >= 3) and (sys.version_info.minor >= 8):
+    import importlib.metadata
+
+    __version__ = importlib.metadata.version("metatensor-learn")
+
+else:
+    from pkg_resources import get_distribution
+
+    __version__ = get_distribution("metatensor-learn").version
```

Comment:
Since we have https://github.com/lab-cosmo/metatensor/pull/425 on the way, I'd be fine only supporting Python 3.7 here